docs/roadmap.md

```diff
@@ -0,0 +1,36 @@
+# `rust-bitcoin` repository roadmap
+
+* ~Release `units v1.0.0-rc0`~
+* Release `primitives v1.0.0-rc0`
+  * Implement script tagging
+  * Add support for consensus encoding to `primitives`
+* Release `bitcoin v0.33.0-rc0`
+* Split out an address crate
+* BIP-32 and BIP-380
+* PSBTv2
+
+## RC cycle
+
+Six month RC cycle for `units` and `primitives` (implies `bitcoin`). Then start cutting the `1.0`
+releases and `bitcoin v0.33`.
```

Comment:
In a3c03adf407483240417df158db7a6230e9cb45e:

This sounds fine but should we have already tried to cut a rc of `bitcoin` alongside the first `units` rc?

Reply:
Do you mean that this text is not clear or do you mean should we already have done this?

The 'implies `bitcoin`' was intended to mean 'we also have to do an RC cycle for bitcoin  e.g. `v0.33-rc.0`'

Reply:
I mean we should've already done it :). The text is clear, it just doesn't match what we've done.